api/envoy/extensions/geoip_providers/maxmind/v3/maxmind.proto

```diff
@@ -23,7 +23,13 @@ option (xds.annotations.v3.file_status).work_in_progress = true;
 // :ref:`anon_db_path <envoy_v3_api_field_extensions.geoip_providers.maxmind.v3.MaxMindConfig.anon_db_path>` must be configured.
```

Comment:
sure, does it make sense to move the warning of "at most one of country_db_path and city_db_path can be configured" here so that these instructions are together? and remove it from line 30 below